x-pack/platform/plugins/shared/ml/public/application/model_management/model_actions.tsx

```diff
@@ -246,20 +263,36 @@ export function useModelActions({
                   ? { adaptive_allocations: modelDeploymentParams.adaptive_allocations }
                   : {}),
               }
-            );
-            await fetchModels();
-          } catch (e) {
-            displayErrorToast(
-              e,
-              i18n.translate('xpack.ml.trainedModels.modelsList.startFailed', {
-                defaultMessage: 'Failed to start "{modelId}"',
-                values: {
-                  modelId: item.model_id,
-                },
-              })
-            );
-            onLoading(false);
-          }
+            )
+            .subscribe({
+              error: (e) => {
+                displayErrorToast(
+                  e,
+                  i18n.translate('xpack.ml.trainedModels.modelsList.startFailed', {
+                    defaultMessage: 'Failed to start "{modelId}"',
+                    values: {
+                      modelId: item.model_id,
+                    },
+                  })
+                );
+              },
+              next: () => {
+                displaySuccessToast({
+                  title: i18n.translate('xpack.ml.trainedModels.modelsList.startSuccess', {
+                    defaultMessage: 'Deployment started',
+                  }),
+                  text: i18n.translate(
+                    'xpack.ml.trainedModels.modelsList.startSuccessDescription',
+                    {
+                      defaultMessage: 'Deployment for "{modelId}" has been started successfully.',
```

Comment:
I'd remove `been` from this message. Or do you know the deployment ID at this point, in which case something like 

```
Deployment .elser_model_2_ingest has started successfully.
```

Reply:
Either
```suggestion
                      defaultMessage: 'Deployment for "{modelId}" has started successfully.',
```
or
```suggestion
                      defaultMessage: 'DeploymentID has started successfully.',
                     
```
The second one would be better if possible, so we wouldn't need to distinguish between model and deployment (`Deployment for modelID`) in the message as it makes the toast harder to skim.

Reply:
Updated with option 2 in https://github.com/elastic/kibana/pull/205699/commits/62b3a61eacf9107f438b64cb1f1d7efaa6302c6a

<img width="365" alt="image" src="https://github.com/user-attachments/assets/6e7a95a0-7af7-4796-b7f4-6b0b7912e8dd" />

Additionally, the error message has been updated to include the deploymentId instead of the modelId

<img width="389" alt="image" src="https://github.com/user-attachments/assets/4db20edb-b5fd-45a8-a661-9890d503a660" />